src/redux/sagas/clients.saga.js

```diff
@@ -30,12 +30,12 @@ export function* fetchAllClients() {
   }
 
 function* updateClient(action) {
-  console.log('In update user', action.payload.id);
+  console.log('In update client', action.payload.id);
   try {
-    yield axios.put(`/api/user`, action.payload);
-    yield put ({type: 'SET_CLIENTS'});
+    yield axios.put(`/api/clients/update/${action.payload.id}`, action.payload);
+    yield put ({type: 'FETCH_CLIENT'});
```

Comment:
is there a reason why Fetch_Client is called here instead of SET_CLIENTS?
I think FETCH_CLIENTS triggers fetchAllClients that gets all clients rather than the individual client that's being updated.